components/ui/window/FloatingRestoreButtons.tsx

```
@@ -0,0 +1,52 @@
+            key={win.id}
+            onClick={() => restoreWindow(win.id)}
+            className={cn(
+              "p-3 rounded-full",
+              "bg-blue-600 hover:bg-blue-700 text-white", // Base style - might want variations
+              "shadow-lg transition-opacity duration-300 ease-in-out",
+              "focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2"
+              // Opacity/pointer-events handled by the conditional rendering above
+            )}
+            aria-label={win.title || `Restore ${win.id}`}
+            title={win.title || `Restore ${win.id}`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)